client/my-sites/domains/domain-management/transfer/transfer-to-other-site/index.jsx

```diff
@@ -59,9 +60,6 @@ export class TransferToOtherSite extends React.Component {
 			site.capabilities.manage_options &&
 			! ( site.jetpack && ! isAtomic ) && // Simple and Atomic sites. Not Jetpack sites.
 			! get( site, 'options.is_domain_only', false ) &&
-			! (
-				this.props.domainsWithPlansOnly && isWpComFreePlan( get( site, 'plan.product_slug' ) )
```

Comment:
I just realised something: won't this also apply to domain registrations?

Instead of being removed completely, I think this check should only be skipped when `this.props.isMapping` is true.

Reply:
Sorry, I should have been clearer here - `domainsWithPlansOnly` is _always_ `false` now. For all users. PWPO took over some of its meaning, but in this case, the check is simply not needed anymore (unless I missed something!). With the improved messaging (based on the PWPO flag), we should have all the bases covered.

Reply:
👍